drivers/packet/provider.go

```diff
@@ -43,20 +45,23 @@ func New(opts ...Option) autoscaler.Provider {
 		p.facility = "ewr1"
 	}
 	if p.os == "" {
-		p.os = "ubuntu_16_04"
+		p.os = "ubuntu_17_10"
```

Comment:
my preference is that we default to `ubuntu_16_04` since it is designated a long term support release, vs 17.10 which I believe only has a 9 month support period.

Reply:
the cloud-config script is not deploying docker properly  so I tried few different OS versions.
I am on the packet slack so once this is resolved will update the PR one last time.

Reply:
got it ... also I just checked and 18.04 is released in 5 weeks. So we can just pick whatever works in the short term, and then update to 18.0.4 when available

Reply:
the cloud-init script is working on gcp/hetzner/amazon on ubuntu 16.04, so it is interesting it does not work on the Packet image. Anything in the cloud-init logs or `journalctl -fu docker.service` that might provide insight?